.travis.yml

```diff
@@ -3,10 +3,19 @@ language: python
 python:
   - 2.6
   - 2.7
+  - 3.4
 
 env:
   - DJANGO_VERSION=1.5
   - DJANGO_VERSION=1.6
+  - DJANGO_VERSION=1.7
+  - DJANGO_VERSION=1.8
+matrix:
+  exclude:
+    - python: 2.6
+      env: DJANGO_VERSION=1.7
+    - python: 2.6
+      env: DJANGO_VERSION=1.8
```

Comment:
Triggered a fresh Travis CI run, and now the tests pass :smile: I think the previous run flaked. Django >=1.5 has python 3 support, so the test matrix should be OK.